docs/network-drivers.md

```diff
@@ -70,20 +70,21 @@ Doing so is not difficult, and it is critical for security.
 ## Switch drivers
 
 At present, all switch drivers shipped with HaaS require that the VLAN
-pool allocator is in use. There are two switch drivers shipped with
+pool allocator is in use. There are three switch drivers shipped with
 HaaS:
 
 * ``haas.ext.switches.dell``, which provides a driver for the Dell
-  Powerconnect 5500 series switches.
+  Powerconnect 5500 series switches and the Dell N3000 series.
```

Comment:
These are separate modules now (the n3000 is in `haas.ext.switches.n3000`), so this needs updating.